--- C:/Git Repositories/Bloodstained-Mods/Base Version Files/1.18/Core/DataTable/PB_DT_ArtsCommandMaster.json	Sun Nov 22 03:54:41 2020
+++ C:/Git Repositories/Bloodstained-Mods/Mods/SimpleTechs/Edits/BloodstainedRotN/Content/Core/DataTable/PB_DT_ArtsCommandMaster.json	Sat Dec  5 21:05:22 2020
@@ -12 +12 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -20 +20 @@
-            "Key": "2",
+            "Key": "6",
@@ -25,5 +24,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -58 +53 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -66 +61 @@
-            "Key": "2",
+            "Key": "6",
@@ -71,5 +65,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -104 +94 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -112 +102 @@
-            "Key": "2",
+            "Key": "6",
@@ -117,5 +106,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -150 +135 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -158 +143 @@
-            "Key": "2",
+            "Key": "6",
@@ -163,5 +147,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -196 +176 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -210,5 +189,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -238 +213 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -252,5 +226,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -280 +250 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -294,5 +263,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -322 +287 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -336,5 +300,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -404 +364 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -416,5 +375,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -444 +399 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -456,5 +410,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -484 +434 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -496,5 +445,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -524 +469 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -536,5 +480,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -564 +504 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -576,5 +515,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -604 +539 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -616,5 +550,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -644 +574 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -656,5 +585,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -684 +609 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -696,5 +620,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -724 +644 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -736,5 +655,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -764 +679 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -776,5 +690,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -804 +714 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -816,5 +725,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -844 +749 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -856,5 +760,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -884 +784 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -896,5 +795,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -924 +819 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -936,5 +830,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -964 +854 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -976,5 +865,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1004 +889 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1016,5 +900,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1044 +924 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1056,5 +935,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1084 +959 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1096,5 +970,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1124 +994 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1136,5 +1005,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1164 +1029 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1176,5 +1040,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1204 +1064 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1216,5 +1075,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1244 +1099 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1256,5 +1110,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1284 +1134 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1296,5 +1145,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1324 +1169 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1336,5 +1180,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1364 +1204 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1376,5 +1215,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1404 +1239 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1416,5 +1250,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1444 +1274 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1456,5 +1285,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1484 +1309 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1496,5 +1320,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1524 +1344 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1536,5 +1355,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1564 +1379 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1576,5 +1390,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1604 +1414 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1616,5 +1425,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1644 +1449 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1656,5 +1460,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1684 +1484 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1696,5 +1495,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1724 +1519 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1736,5 +1530,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1764 +1554 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1776,5 +1565,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1804 +1589 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1816,5 +1600,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1844 +1624 @@
-      "Expertise": 120,
+      "Expertise": 10,
@@ -1856,5 +1635,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1884 +1659 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -1919 +1694 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -1926 +1701 @@
-            "Key": "2",
+            "Key": "8",
@@ -1931,10 +1705,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -1964 +1729 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -1971 +1736 @@
-            "Key": "2",
+            "Key": "8",
@@ -1976,10 +1740,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2009 +1764 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2016 +1771 @@
-            "Key": "2",
+            "Key": "8",
@@ -2021,10 +1775,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2054 +1799 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2061 +1806 @@
-            "Key": "2",
+            "Key": "8",
@@ -2066,10 +1810,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2099 +1834 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2106 +1841 @@
-            "Key": "2",
+            "Key": "8",
@@ -2111,10 +1845,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2144 +1869 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2151 +1876 @@
-            "Key": "2",
+            "Key": "8",
@@ -2156,10 +1880,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2189 +1904 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2196 +1911 @@
-            "Key": "2",
+            "Key": "8",
@@ -2201,10 +1915,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2234 +1939 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2241 +1946 @@
-            "Key": "2",
+            "Key": "8",
@@ -2246,10 +1950,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2279 +1974 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2315 +2010 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -2327,5 +2021,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2355 +2045 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -2367,5 +2056,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2395 +2080 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -2407,5 +2091,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2435 +2115 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2442 +2122 @@
-            "Key": "2",
+            "Key": "8",
@@ -2447,10 +2126,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2480 +2150 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2487 +2157 @@
-            "Key": "2",
+            "Key": "8",
@@ -2492,10 +2161,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2525 +2185 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2532 +2192 @@
-            "Key": "2",
+            "Key": "8",
@@ -2537,10 +2196,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2570 +2220 @@
-      "Expertise": 500,
+      "Expertise": 10,
@@ -2620 +2270 @@
-      "Expertise": 500,
+      "Expertise": 10,
@@ -2670 +2320 @@
-      "Expertise": 500,
+      "Expertise": 10,
@@ -2720 +2370 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -2760 +2410 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -2800 +2450 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -2812,5 +2461,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2840 +2485 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -2852,5 +2496,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2915 +2555 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2922 +2562 @@
-            "Key": "2",
+            "Key": "8",
@@ -2927,5 +2566,0 @@
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2937,5 +2571,0 @@
-            "Key": "6",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2960 +2590 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -2967 +2597 @@
-            "Key": "2",
+            "Key": "8",
@@ -2972,5 +2601,0 @@
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -2982,5 +2606,0 @@
-            "Key": "6",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3005 +2625 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -3042 +2662 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -3079 +2699 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3093,5 +2712,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3121 +2736 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3135,5 +2749,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3163 +2773 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3175,5 +2784,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3203 +2808 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3215,5 +2819,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3243 +2843 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3255,5 +2854,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3283 +2878 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -3290 +2885 @@
-            "Key": "2",
+            "Key": "8",
@@ -3295,10 +2889,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3328 +2913 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -3335 +2920 @@
-            "Key": "2",
+            "Key": "8",
@@ -3340,10 +2924,0 @@
-            "Key": "1",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "4",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3373 +2948 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3408 +2983 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3443 +3018 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -3483 +3058 @@
-      "Expertise": 80,
+      "Expertise": 10,
@@ -3523 +3098 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3530 +3105 @@
-            "Key": "4",
+            "Key": "8",
@@ -3535,10 +3109,0 @@
-            "Key": "2",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "6",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3568 +3133 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3575 +3140 @@
-            "Key": "4",
+            "Key": "8",
@@ -3580,10 +3144,0 @@
-            "Key": "2",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "6",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3613 +3168 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3620 +3175 @@
-            "Key": "4",
+            "Key": "8",
@@ -3625,10 +3179,0 @@
-            "Key": "2",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
-            "Key": "6",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3658 +3203 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3670,5 +3214,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3698 +3238 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3710,5 +3249,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3738 +3273 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3750,5 +3284,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3778 +3308 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3813 +3343 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3848 +3378 @@
-      "Expertise": 50,
+      "Expertise": 10,
@@ -3931,5 +3460,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -3971,5 +3495,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -4011,5 +3530,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
@@ -4091,5 +3605,0 @@
-            "Key": "3",
-            "Time": 0.2,
-            "UTTypes": "NameProperty,FloatProperty"
-          },
-          {
