--- C:/Git Repositories/Bloodstained-Mods/Base Version Files/1.18/L10N/en/Core/StringTable/PBMasterStringTable.json	Wed Dec  2 02:39:04 2020
+++ C:/Git Repositories/Bloodstained-Mods/Mods/RebalancedItems/Edits/BloodstainedRotN/Content/L10N/en/Core/StringTable/PBMasterStringTable.json	Wed Dec  2 02:40:35 2020
@@ -180 +180 @@
-    "ITEM_EXPLAIN_CowboyHat": "A hat worn by cowboys.\r\nIncreases the power of firearms.",
+    "ITEM_EXPLAIN_CowboyHat": "A hat worn by cowboys.\r\nIncreases the power of firearms by 50%.",
@@ -188 +188 @@
-    "ITEM_EXPLAIN_Westernhat": "A hat that doubles the ammunition you use but\r\nmakes firearms stronger.",
+    "ITEM_EXPLAIN_Westernhat": "A hat that doubles the ammunition you use but\r\nmakes firearms deal 2x damage.",
@@ -191 +191 @@
-    "ITEM_EXPLAIN_Magushat": "A witch's hat that aids concentration and hastens \r\nmagic recovery.",
+    "ITEM_EXPLAIN_Magushat": "A witch's hat that aids concentration and hastens \r\nmagic recovery by 20%.",
@@ -220 +220 @@
-    "ITEM_EXPLAIN_IronMansApron": "An apron imbued with the soul of a departed chef.",
+    "ITEM_EXPLAIN_IronMansApron": "An apron imbued with the soul of a departed chef. 10% chance to get 1 extra food.",
@@ -225 +225 @@
-    "ITEM_EXPLAIN_Shovelarmorsarmor": "Armor formerly worn by a certain valiant knight.",
+    "ITEM_EXPLAIN_Shovelarmorsarmor": "Armor formerly worn by a certain valiant knight. Turns you into a Shovel Knight.",
@@ -236 +236 @@
-    "ITEM_EXPLAIN_Assassinvest": "Assassin's garb that increases the speed of your\r\nattacks.",
+    "ITEM_EXPLAIN_Assassinvest": "Assassin's garb that increases the speed of your\r\nattacks by 10%.",
@@ -255 +255 @@
-    "ITEM_EXPLAIN_Gebelsglasses": "Hand-me-downs from Johannes.\r\nA reminder of days that are now lost.",
+    "ITEM_EXPLAIN_Gebelsglasses": "Hand-me-downs from Johannes.\r\nA reminder of days that are now lost. Stop magic depletion.",
@@ -273,4 +273,4 @@
-    "ITEM_EXPLAIN_Blackbelt": "A belt worn by Eastern masters of the martial arts.\r\nIncreases attack speed.",
-    "ITEM_EXPLAIN_Striderbelt": "A belt made to withstand rough movement.\r\nImproves sliding ability.",
-    "ITEM_EXPLAIN_Moonring": "A belt with a moon design that improves your\r\nbackstep ability.",
-    "ITEM_EXPLAIN_Speedstar": "A strange belt that compels wearers to run. \r\nIncreases movement speed.",
+    "ITEM_EXPLAIN_Blackbelt": "A belt worn by Eastern masters of the martial arts.\r\nIncreases attack speed by 10%.",
+    "ITEM_EXPLAIN_Striderbelt": "A belt made to withstand rough movement.\r\nImproves sliding distance by 20%.",
+    "ITEM_EXPLAIN_Moonring": "A belt with a moon design that improves your\r\nbackstep distance by 20%.",
+    "ITEM_EXPLAIN_Speedstar": "A strange belt that compels wearers to run. \r\nIncreases movement speed by 15%.",
@@ -278,3 +278,3 @@
-    "ITEM_EXPLAIN_SkullNecklace": "A necklace that amplifies the power of shards at\r\nincreased magic cost. ",
-    "ITEM_EXPLAIN_DemonNecklace": "A demon necklace that eliminates vulnerability\r\nwhen using shards. ",
-    "ITEM_EXPLAIN_WarlocksNecklace": "The beloved necklace of a great sorcerer.\r\nReduces magic depletion.",
+    "ITEM_EXPLAIN_SkullNecklace": "A necklace that amplifies the power of shards by 50%\r\nat increased doubled magic cost.",
+    "ITEM_EXPLAIN_DemonNecklace": "A demon necklace that eliminates vulnerability\r\nwhen using shards (20% extra speed). ",
+    "ITEM_EXPLAIN_WarlocksNecklace": "The beloved necklace of a great sorcerer.\r\nReduces magic depletion by 20%.",
@@ -282,3 +282,3 @@
-    "ITEM_EXPLAIN_Flamering": "A ring blessed by a fire god.\r\nStrengthens fire abilities.",
-    "ITEM_EXPLAIN_Icering": "A ring blessed by an ice god.\r\nStrengthens ice abilities.",
-    "ITEM_EXPLAIN_Thunderring": "A ring blessed by a thunder god.\r\nStrengthens thunder abilities.",
+    "ITEM_EXPLAIN_Flamering": "A ring blessed by a fire god.\r\nStrengthens fire abilities by 20%.",
+    "ITEM_EXPLAIN_Icering": "A ring blessed by an ice god.\r\nStrengthens ice abilities by 20%.",
+    "ITEM_EXPLAIN_Thunderring": "A ring blessed by a thunder god.\r\nStrengthens thunder abilities by 20%.",
@@ -287,9 +287,9 @@
-    "ITEM_EXPLAIN_AssassinsRing": "A ring that slightly increases damage and\r\nfrequency of critical hits.",
-    "ITEM_EXPLAIN_LethalityRing": "A ring that increases the damage you inflict\r\nwith critical hits.",
-    "ITEM_EXPLAIN_Criticalring": "A ring that increases the likelihood of critical hits.",
-    "ITEM_EXPLAIN_CutpursesRing": "A ring that makes enemies more likely to drop gold.",
-    "ITEM_EXPLAIN_PlunderersRing": "A ring that increases the rate of item drops.",
-    "ITEM_EXPLAIN_RingofSolomon": "A ring that increases how often enemies yield shards.",
-    "ITEM_EXPLAIN_WeightedRing": "A heavy ring that increases your experience intake.",
-    "ITEM_EXPLAIN_UnicornRing": "A ring that amplifies the restorative properties of\r\nitems you use.",
-    "ITEM_EXPLAIN_RoseRing": "A ring that increases the amount of magic restored\r\nby mana roses.",
+    "ITEM_EXPLAIN_AssassinsRing": "A ring that slightly increases damage (10%) and\r\nfrequency (5%) of critical hits.",
+    "ITEM_EXPLAIN_LethalityRing": "A ring that increases the damage you inflict\r\nwith critical hits by 20%.",
+    "ITEM_EXPLAIN_Criticalring": "A ring that increases the likelihood of critical hits by 10%.",
+    "ITEM_EXPLAIN_CutpursesRing": "A ring that makes enemies more likely to drop gold by 10%.",
+    "ITEM_EXPLAIN_PlunderersRing": "A ring that increases the rate of item drops by 25%.",
+    "ITEM_EXPLAIN_RingofSolomon": "A ring that increases how often enemies yield shards by 10%.",
+    "ITEM_EXPLAIN_WeightedRing": "A heavy ring that increases your experience intake by 25%.",
+    "ITEM_EXPLAIN_UnicornRing": "A ring that amplifies the restorative properties of\r\nitems you use by 20%.",
+    "ITEM_EXPLAIN_RoseRing": "A ring that increases the amount of magic restored\r\nby mana roses by 25%.",
@@ -300,4 +300,4 @@
-    "ITEM_EXPLAIN_Walkerking": "A ring that increases attack power in proportion to\r\nhow much of the map you have completed.",
-    "ITEM_EXPLAIN_Goldmanring": "A ring that increases attack power in proportion to\r\nyour wealth.",
-    "ITEM_EXPLAIN_Silvermanring": "A ring that increases attack power in proportion to\r\nthe number of enemies you have slain.",
-    "ITEM_EXPLAIN_Adversityring": "A ring that causes status inflictions to increase\r\nyour power.",
+    "ITEM_EXPLAIN_Walkerking": "A ring that increases attack power in proportion to\r\nhow much of the map you have completed. (20%)",
+    "ITEM_EXPLAIN_Goldmanring": "A ring that increases attack power in proportion to\r\nyour wealth. (20%)",
+    "ITEM_EXPLAIN_Silvermanring": "A ring that increases attack power in proportion to\r\nthe number of enemies you have slain. (20%)",
+    "ITEM_EXPLAIN_Adversityring": "A ring that causes status inflictions to increase\r\nyour power by 20%.",
@@ -310 +310 @@
-    "ITEM_EXPLAIN_Outlinestyle": "A scarf that makes you quick on the draw and \r\nincreases firing rate.",
+    "ITEM_EXPLAIN_Outlinestyle": "A scarf that makes you quick on the draw and \r\nincreases firing rate by 10%.",
@@ -2194 +2194 @@
-    "ITEM_EXPLAIN_Liddyl": "A sword once used to remove a dragon's heart.",
+    "ITEM_EXPLAIN_Liddyl": "A sword once used to remove a dragon's heart. Hit 3 times.",
@@ -2273 +2273 @@
-    "ITEM_EXPLAIN_SacredSword": "A greatsword containing the power of dawn.",
+    "ITEM_EXPLAIN_SacredSword": "A katana containing the power of dawn.",
@@ -2285,2 +2285,2 @@
-    "ITEM_EXPLAIN_LightSaber": "6F20636B6E7077206772646C7A207A77\r\n6B6A207075707365756C2078726B7674",
-    "ITEM_EXPLAIN_JodoSwordLight": "A sword that purifies evil with its divine light.",
+    "ITEM_EXPLAIN_LightSaber": "A light saber with immense power.",
+    "ITEM_EXPLAIN_JodoSwordLight": "A dagger that purifies evil with its divine light.",
@@ -2299 +2299 @@
-    "ITEM_EXPLAIN_ValkyrieSword": "A warmaiden's weapon of choice.",
+    "ITEM_EXPLAIN_ValkyrieSword": "A warmaiden's weapon of choice. Secret art.",
