--- C:/Git Repositories/Bloodstained-Mods/Base Version Files/1.18/Core/DataTable/PB_DT_WeaponMaster.json	Sun Nov 22 03:55:00 2020
+++ C:/Git Repositories/Bloodstained-Mods/Mods/RebalancedItems/Edits/BloodstainedRotN/Content/Core/DataTable/PB_DT_WeaponMaster.json	Sun Nov 29 19:11:20 2020
@@ -18 +18 @@
-      "MeleeAttack": 8,
+      "MeleeAttack": 10,
@@ -68 +68 @@
-      "MeleeAttack": 12,
+      "MeleeAttack": 14,
@@ -118 +118 @@
-      "MeleeAttack": 14,
+      "MeleeAttack": 16,
@@ -168 +168 @@
-      "MeleeAttack": 17,
+      "MeleeAttack": 19,
@@ -218 +218 @@
-      "MeleeAttack": 21,
+      "MeleeAttack": 23,
@@ -268 +268 @@
-      "MeleeAttack": 24,
+      "MeleeAttack": 26,
@@ -318,2 +318,2 @@
-      "MeleeAttack": 28,
-      "MagicAttack": 28,
+      "MeleeAttack": 30,
+      "MagicAttack": 30,
@@ -368 +368 @@
-      "MeleeAttack": 38,
+      "MeleeAttack": 40,
@@ -419 +419 @@
-      "MagicAttack": 0,
+      "MagicAttack": 45,
@@ -468 +468 @@
-      "MeleeAttack": 8,
+      "MeleeAttack": 10,
@@ -518 +518 @@
-      "MeleeAttack": 14,
+      "MeleeAttack": 16,
@@ -568 +568 @@
-      "MeleeAttack": 19,
+      "MeleeAttack": 21,
@@ -618 +618 @@
-      "MeleeAttack": 16,
+      "MeleeAttack": 18,
@@ -668 +668 @@
-      "MeleeAttack": 24,
+      "MeleeAttack": 27,
@@ -718 +718 @@
-      "MeleeAttack": 29,
+      "MeleeAttack": 32,
@@ -768,2 +768,2 @@
-      "MeleeAttack": 30,
-      "MagicAttack": 30,
+      "MeleeAttack": 33,
+      "MagicAttack": 33,
@@ -814 +814 @@
-      "WeaponEffectAttribute": "None",
+      "WeaponEffectAttribute": "Darkness",
@@ -819 +819 @@
-      "MagicAttack": 0,
+      "MagicAttack": 45,
@@ -827 +827 @@
-      "DAR": false,
+      "DAR": true,
@@ -1864 +1864 @@
-      "WeaponEffectAttribute": "None",
+      "WeaponEffectAttribute": "Darkness",
@@ -1869 +1869 @@
-      "MagicAttack": 0,
+      "MagicAttack": 20,
@@ -1877 +1877 @@
-      "DAR": false,
+      "DAR": true,
@@ -1918 +1918 @@
-      "MeleeAttack": 53,
+      "MeleeAttack": 54,
@@ -1921,2 +1921,2 @@
-      "DAG": false,
-      "TOT": false,
+      "DAG": true,
+      "TOT": true,
@@ -2121,2 +2121,2 @@
-      "DAG": false,
-      "TOT": false,
+      "DAG": true,
+      "TOT": true,
@@ -2168 +2168 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 17,
@@ -2218 +2218 @@
-      "MeleeAttack": 20,
+      "MeleeAttack": 22,
@@ -2268 +2268 @@
-      "MeleeAttack": 30,
+      "MeleeAttack": 33,
@@ -2318,2 +2318,2 @@
-      "MeleeAttack": 32,
-      "MagicAttack": 32,
+      "MeleeAttack": 36,
+      "MagicAttack": 36,
@@ -2368 +2368 @@
-      "MeleeAttack": 45,
+      "MeleeAttack": 50,
@@ -2418,2 +2418,2 @@
-      "MeleeAttack": 50,
-      "MagicAttack": 50,
+      "MeleeAttack": 55,
+      "MagicAttack": 55,
@@ -2468,2 +2468,2 @@
-      "MeleeAttack": 52,
-      "MagicAttack": 52,
+      "MeleeAttack": 58,
+      "MagicAttack": 58,
@@ -4018 +4018 @@
-      "MeleeAttack": 13,
+      "MeleeAttack": 14,
@@ -4068 +4068 @@
-      "MeleeAttack": 18,
+      "MeleeAttack": 20,
@@ -4118 +4118 @@
-      "MeleeAttack": 22,
+      "MeleeAttack": 24,
@@ -4168 +4168 @@
-      "MeleeAttack": 27,
+      "MeleeAttack": 30,
@@ -4218 +4218 @@
-      "MeleeAttack": 29,
+      "MeleeAttack": 32,
@@ -4268 +4268 @@
-      "MeleeAttack": 33,
+      "MeleeAttack": 35,
@@ -4318 +4318 @@
-      "MeleeAttack": 36,
+      "MeleeAttack": 39,
@@ -4518,2 +4518,2 @@
-      "MeleeAttack": 46,
-      "MagicAttack": 46,
+      "MeleeAttack": 58,
+      "MagicAttack": 58,
@@ -4568 +4568 @@
-      "MeleeAttack": 13,
+      "MeleeAttack": 15,
@@ -4618 +4618 @@
-      "MeleeAttack": 20,
+      "MeleeAttack": 22,
@@ -4668,2 +4668,2 @@
-      "MeleeAttack": 27,
-      "MagicAttack": 27,
+      "MeleeAttack": 30,
+      "MagicAttack": 30,
@@ -4718 +4718 @@
-      "MeleeAttack": 35,
+      "MeleeAttack": 39,
@@ -4768 +4768 @@
-      "MeleeAttack": 32,
+      "MeleeAttack": 36,
@@ -4818 +4818 @@
-      "MeleeAttack": 45,
+      "MeleeAttack": 50,
@@ -4868,2 +4868,2 @@
-      "MeleeAttack": 50,
-      "MagicAttack": 50,
+      "MeleeAttack": 55,
+      "MagicAttack": 55,
@@ -4918,2 +4918,2 @@
-      "MeleeAttack": 52,
-      "MagicAttack": 52,
+      "MeleeAttack": 58,
+      "MagicAttack": 58,
@@ -4968 +4968 @@
-      "MeleeAttack": 5,
+      "MeleeAttack": 6,
@@ -5068 +5068 @@
-      "MeleeAttack": 12,
+      "MeleeAttack": 14,
@@ -5118 +5118 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 18,
@@ -5168 +5168 @@
-      "MeleeAttack": 20,
+      "MeleeAttack": 22,
@@ -5218 +5218 @@
-      "MeleeAttack": 23,
+      "MeleeAttack": 26,
@@ -5268 +5268 @@
-      "MeleeAttack": 25,
+      "MeleeAttack": 30,
@@ -5318 +5318 @@
-      "MeleeAttack": 26,
+      "MeleeAttack": 32,
@@ -5418,2 +5418,2 @@
-      "MeleeAttack": 23,
-      "MagicAttack": 23,
+      "MeleeAttack": 29,
+      "MagicAttack": 29,
@@ -5468,2 +5468,2 @@
-      "MeleeAttack": 46,
-      "MagicAttack": 46,
+      "MeleeAttack": 58,
+      "MagicAttack": 58,
@@ -5518,2 +5518,2 @@
-      "MeleeAttack": 67,
-      "MagicAttack": 67,
+      "MeleeAttack": 84,
+      "MagicAttack": 84,
@@ -6318,2 +6318,2 @@
-      "MeleeAttack": 16,
-      "MagicAttack": 16,
+      "MeleeAttack": 20,
+      "MagicAttack": 20,
@@ -6368,2 +6368,2 @@
-      "MeleeAttack": 32,
-      "MagicAttack": 32,
+      "MeleeAttack": 40,
+      "MagicAttack": 40,
@@ -6418,2 +6418,2 @@
-      "MeleeAttack": 46,
-      "MagicAttack": 46,
+      "MeleeAttack": 58,
+      "MagicAttack": 58,
@@ -6483,6 +6483,6 @@
-      "ScaleX": 1.0,
-      "ScaleY": 1.0,
-      "ScaleZ": 1.0,
-      "EffScaleX": 1.25,
-      "EffScaleY": 1.05,
-      "EffScaleZ": 1.25,
+      "ScaleX": 1.4,
+      "ScaleY": 1.4,
+      "ScaleZ": 1.4,
+      "EffScaleX": 1.65,
+      "EffScaleY": 1.45,
+      "EffScaleZ": 1.65,
@@ -6533,6 +6533,6 @@
-      "ScaleX": 1.0,
-      "ScaleY": 1.0,
-      "ScaleZ": 1.0,
-      "EffScaleX": 1.25,
-      "EffScaleY": 1.05,
-      "EffScaleZ": 1.25,
+      "ScaleX": 1.4,
+      "ScaleY": 1.4,
+      "ScaleZ": 1.4,
+      "EffScaleX": 1.65,
+      "EffScaleY": 1.45,
+      "EffScaleZ": 1.65,
@@ -6583,6 +6583,6 @@
-      "ScaleX": 1.0,
-      "ScaleY": 1.0,
-      "ScaleZ": 1.0,
-      "EffScaleX": 1.25,
-      "EffScaleY": 1.05,
-      "EffScaleZ": 1.25,
+      "ScaleX": 1.4,
+      "ScaleY": 1.4,
+      "ScaleZ": 1.4,
+      "EffScaleX": 1.65,
+      "EffScaleY": 1.45,
+      "EffScaleZ": 1.65,
@@ -6618,2 +6618,2 @@
-      "MeleeAttack": 11,
-      "MagicAttack": 11,
+      "MeleeAttack": 16,
+      "MagicAttack": 16,
@@ -6668,2 +6668,2 @@
-      "MeleeAttack": 23,
-      "MagicAttack": 23,
+      "MeleeAttack": 32,
+      "MagicAttack": 32,
@@ -6718,2 +6718,2 @@
-      "MeleeAttack": 34,
-      "MagicAttack": 34,
+      "MeleeAttack": 46,
+      "MagicAttack": 46,
@@ -7068,2 +7068,2 @@
-      "MeleeAttack": 18,
-      "MagicAttack": 18,
+      "MeleeAttack": 20,
+      "MagicAttack": 20,
@@ -7118,2 +7118,2 @@
-      "MeleeAttack": 36,
-      "MagicAttack": 36,
+      "MeleeAttack": 40,
+      "MagicAttack": 40,
@@ -7168,2 +7168,2 @@
-      "MeleeAttack": 52,
-      "MagicAttack": 52,
+      "MeleeAttack": 58,
+      "MagicAttack": 58,
@@ -7368,2 +7368,2 @@
-      "MeleeAttack": 23,
-      "MagicAttack": 23,
+      "MeleeAttack": 29,
+      "MagicAttack": 29,
@@ -7418,2 +7418,2 @@
-      "MeleeAttack": 46,
-      "MagicAttack": 46,
+      "MeleeAttack": 58,
+      "MagicAttack": 58,
@@ -7468,2 +7468,2 @@
-      "MeleeAttack": 67,
-      "MagicAttack": 67,
+      "MeleeAttack": 84,
+      "MagicAttack": 84,
@@ -7668,2 +7668,2 @@
-      "MeleeAttack": 17,
-      "MagicAttack": 17,
+      "MeleeAttack": 23,
+      "MagicAttack": 23,
@@ -7718,2 +7718,2 @@
-      "MeleeAttack": 35,
-      "MagicAttack": 35,
+      "MeleeAttack": 46,
+      "MagicAttack": 46,
@@ -7768,2 +7768,2 @@
-      "MeleeAttack": 50,
-      "MagicAttack": 50,
+      "MeleeAttack": 67,
+      "MagicAttack": 67,
@@ -8268 +8268 @@
-      "MeleeAttack": 18,
+      "MeleeAttack": 20,
@@ -8318 +8318 @@
-      "MeleeAttack": 36,
+      "MeleeAttack": 40,
@@ -8368 +8368 @@
-      "MeleeAttack": 52,
+      "MeleeAttack": 58,
@@ -8418 +8418 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8468 +8468 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8470,2 +8470,2 @@
-      "ZAN": true,
-      "DAG": false,
+      "ZAN": false,
+      "DAG": true,
@@ -8518 +8518 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8520 +8520 @@
-      "ZAN": true,
+      "ZAN": false,
@@ -8522 +8522 @@
-      "TOT": false,
+      "TOT": true,
@@ -8564 +8564 @@
-      "WeaponEffectAttribute": "None",
+      "WeaponEffectAttribute": "Fire",
@@ -8568 +8568 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8570 +8570 @@
-      "ZAN": true,
+      "ZAN": false,
@@ -8573 +8573 @@
-      "FLA": false,
+      "FLA": true,
@@ -8614 +8614 @@
-      "WeaponEffectAttribute": "None",
+      "WeaponEffectAttribute": "Ice",
@@ -8618 +8618 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8620 +8620 @@
-      "ZAN": true,
+      "ZAN": false,
@@ -8624 +8624 @@
-      "ICE": false,
+      "ICE": true,
@@ -8668 +8668 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8670 +8670 @@
-      "ZAN": true,
+      "ZAN": false,
@@ -8675 +8675 @@
-      "LIG": false,
+      "LIG": true,
@@ -8718 +8718 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8720 +8720 @@
-      "ZAN": true,
+      "ZAN": false,
@@ -8726 +8726 @@
-      "HOL": false,
+      "HOL": true,
@@ -8768 +8768 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 58,
@@ -8770 +8770 @@
-      "ZAN": true,
+      "ZAN": false,
@@ -8777 +8777 @@
-      "DAR": false,
+      "DAR": true,
@@ -8818 +8818 @@
-      "MeleeAttack": 15,
+      "MeleeAttack": 17,
